exporter/sumologicexporter/source_format_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sumologicexporter
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func getTestSourceFormat(t *testing.T, template string) sourceFormat {
+	r, err := regexp.Compile(`\%\{(\w+)\}`)
```

Comment:
```suggestion
	r, err := regexp.Compile(sourceRegex)
```

Reply:
It won't catch accidental change of the regex const